e2e/utils/logout.ts

```diff
@@ -0,0 +1,15 @@
+import { Page, expect } from '@playwright/test';
+import translations from '../../client/i18n/locales/english/translations.json';
+
+export async function signout(page: Page) {
+  await page.goto('/');
```

Comment:
Given that going to a page is pretty slow and we're likely to use the helper multiple times, I would recommend making an api request in the helper.

This function would make a good spec, though.

It's analogous to what I did in https://github.com/freeCodeCamp/freeCodeCamp/pull/54752/